staking-miner-playground/runtime/src/lib.rs

```diff
@@ -450,15 +461,41 @@ parameter_types! {
 
 	// miner configs
 	pub const ElectionUnsignedPriority: TransactionPriority = StakingUnsignedPriority::get() - 1u64;
-	pub MinerMaxWeight: Weight = Perbill::from_rational(8u32, 10) *
-		<Runtime as frame_system::Config>::BlockWeights::get()
-		.get(DispatchClass::Normal)
-		.max_total
-		.unwrap();
-	pub MinerMaxLength: u32 = Perbill::from_rational(8u32, 10) *
-		*(<<Runtime as frame_system::Config>::BlockLength as Get<limits::BlockLength>>::get())
-		.max
-		.get(DispatchClass::Normal);
+
+	// This is a hack to get the number of validators, candidates and nominators
+	// used by node which uses the same env flags as the chain spec builder in the node crate.
+	Nominators: u32 = option_env!("N").unwrap_or("1000").parse().expect("env variable `N` must be number");
```

Comment:
Nominators and Candidates are not used anymore.

Remove or should we configure something else in staking to work properly?

Reply:
nothing let's remove it